src/app/components/speakers/speakers.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container justify-content-center">
+    <div class="row">
+        <div *ngFor="let speaker of (showAll ? (allSpeakers$ | async) : (allSpeakers$ | async | slice:0:3))"
+            class="col-md-4 mb-4">
+
+
+            <div class="custom-card" (click)="showSpeakerDetails(speaker.code)">
```

Comment:
this should be a link (routerLink) linking to a detail page with speaker.code as parameter to fetch its details on that page.